remove connections from workload to itself regardless existence of replicas (#152)
np-guard/netpol-analyzer: README.md

### List command
```
Lists all allowed connections based on the workloads and network policies defined.

> Excluding self looped connections such as trivial connections from a peer to itself,
> or connections between different replicas of same workload.

Reviewer: ```suggestion
Connections between workload to itself are excluded from the output.
```
